test/RelationTest.php

```diff
@@ -6,17 +6,33 @@
 
 class RelationTest extends DatabaseTestCase
 {
+    private string $books;
+    private string $mixedCaseField;
+
+    public function setUp(string $connection_name = null): void
+    {
+        if (parent::connect('pgsql')) {
```

Comment:
This is... well, kind of gross. This library is supposed to be an abstraction and hide away the details of whichever database technology you're connecting to. When special cases like this start creeping into the tests, it's  a sign that something is breaking down. 

If you can find a way to isolate all this stuff in `PgsqlAdapter` then I won't object, but otherwise I think I was happier with the state of things before you introduced `mixedCaseField`.

Reply:
Another idea would be to teach `PgsqlAdapter` how to auto-quote column names internally.

Reply:
Taught SQLBuilder how to auto-quote column names